src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,692 @@ PICS:
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "Capacity"
+                value: 7
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneList"
+                value: [0x01]
 
     - label:
           "TH sends a RemoveScene command to DUT with the GroupID field set to
           0x0001 and the SceneID field set to 0x01."
       PICS: S.S.C02.Rsp
-      verification: |
-          DUT sends a RemoveSceneResponse command to TH with the Status field set to 0x00 (SUCCESS), the GroupID field set to 0x0001 and the SceneID field set to 0x01.
-      disabled: true
+      command: "RemoveScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
 
     - label:
           "TH sends a GetSceneMembership command to DUT with the GroupID field
           set to 0x0001."
       PICS: S.S.C06.Rsp
-      verification: |
-          DUT sends a GetSceneMembershipResponse command to TH with the Status field set to 0x00 (SUCCESS), the Capacity field set to one of the values SC0, 0xfe or null, the GroupID field set to 0x0001 and the SceneCount field set to 0x00.
-      disabled: true
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "Capacity"
+                value: 8
+              - name: "GroupID"
+                value: 0x0001
 
     - label:
           "TH configures AC2 on DUT for all implemented application clusters
           supporting scenes."
-      verification: |
-          DUT is configured with AC2 which is different from AC1.
-      disabled: true
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 200
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
 
     - label:
           "TH sends a RecallScene command to DUT with the GroupID field set to
           0x0001 and the SceneID field set to 0x01."
       PICS: S.S.C05.Rsp
-      verification: |
-          If a status response is expected, DUT sends a response to TH with the Status field equal to 0x8b (NOT_FOUND).
-      disabled: true
+      command: "RecallScene"
```

Comment:
How is this testing for NOT_FOUND?  I don't think it is.

Reply:
In the current state, as there is no RecallSceneResponse function, 
`      response:
          values:
              - name: "Status"
                value: 0x001`
Causes an error on generation. My conclusion was that a status response was not expected. What mechanism could I use to access the response from yaml?

Reply:
```
      command: "RecallScened"
      arguments:
          values:
...
      response:
          error: NOT_FOUND
```

Reply:
Thank you. Applied in latest commit.